src/component/Clock/Clock.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+type State = {
+  time: string,
+};
+
+type Props = {
+  clockName: number | string,
+};
+
+export class Clock extends React.Component<Props, State> {
+  timerId: NodeJS.Timer | null = null ;
+
+  state: State = {
+    time: new Date().toLocaleTimeString(),
+  };
+
+  componentDidMount() {
+    this.timerId = setInterval(() => {
+      this.updateTime();
```

Comment:
On this method, you already set 1000 seconds to the interval. You don't need to set a new interval on the updateTime method